### PR TITLE
grains: clarify square numbering convention (1-based, not 0-based)

### PR DESCRIPTION
**Pull request by LarryRuane**
Also clarify that the task is to calculate the number of grains on a _given_ square, not on "each" square.

### CODE REVIEW

**Comment by bitfield** [comment]
I think this is a good idea, @LarryRuane. I'm not sure about this exact wording:

> There are 64 squares on a chessboard, numbered 1 through 64.

That makes it sound as though the squares on a chessboard _are_ numbered, which they aren't (at least, not on my chessboard). In so far as they have identifiers, they're identified as `a1`, `h8`, and so on. However, I don't think that's the point you were making.

It sounds like you want to clarify the specification of the `square` property to make it clear that the test cases assume that the "first" square, as defined by the wily servant of the problem description, is considered square 1, and the "last" square is addressed as square 64.

How about this wording instead:

> Write code that shows:
> - How many grains there would be on a given square (where square 1 has one grain, square 2 has two grains, and so on)
>- The total number of grains on the board

**Comment by LarryRuane** [comment]
I see your point perfectly. In the second paragraph, we do say:
> One grain on the first square of a chess board. Two grains on the next. Four on the third, and so on.

(It's just that "first" could mean square 0 or square 1.) So maybe what you're suggesting is slightly redundant. But we can definitely do that. Or, what do you think of:
> There are 64 squares on a chessboard, which we will label 1 through 64.

(Or maybe "identify" instead of "label"?) But either way is okay with me; let me know what you think and I'll force-push an update. (Unless you don't like force-push here?)

The only reason I created this PR, by the way, is because when I'm mentoring solutions, I found myself being "off-by-one" confused, and I actually had to check the test driver to see how squares were identified. But if that's only me, we don't have to merge this PR. But thank you for your quick reply!

**Comment by bitfield** [comment]
> There are 64 squares on a chessboard, which we will label 1 through 64.

This is good, but I can see someone being confused by it, thinking maybe they need to create some kind of construct in their code that represents the board. That's why I suggested:

> How many grains there would be on a given square (where square 1 has one grain, square 2 has two grains, and so on)

to make it clear that "square 1" only has meaning for that bit of code which determines the number of grains on a given square.

Wording these problem descriptions clearly, accurately, unambiguously, and helpfully is surprisingly challenging! We often need to go through a few iterations, so feel free to amend your commit and force-push it if you come up with a better form of words. Others may want to chime in too.

File: exercises/grains/description.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ king loved chess, the servant told the king he would like to have grains
 of wheat. One grain on the first square of a chess board. Two grains on
 the next. Four on the third, and so on.
 
-There are 64 squares on a chessboard.
+There are 64 squares on a chessboard, numbered 1 through 64.
```
Comment (rpottsoh) [inline exercises/grains/description.md]: Perhaps worded like this:
```suggestion
There are 64 squares on a chessboard starting at one.
```

**Comment by bitfield** [comment]
Infuriatingly difficult, isn't it? I should know, I write technical books for a living.

**Comment by bitfield** [comment]
Looks good to me @LarryRuane!
